resources/views/entities/pages/abilities/reorder/_reorder.blade.php

```diff
@@ -0,0 +1,50 @@
+<?php /** @var \App\Models\TimelineEra[] $abilities */?>
+{!! Form::open([
+        'route' => ['abilities.reorder-save', $entity],
+        'method' => 'POST',
+    ]) !!}
+<div class="box box-solid box-entity-story-reorder">
+    <div class="box-header">
+        <h3 class="box-title">
+            {{ __('abilities.reorder.title') }}
+        </h3>
+    </div>
+    <div class="box-body">
+
+    @foreach($parents as $key => $parent)
+
+        <h3 class="box-title">
```

Comment:
`box-title` only works in a `box-header` parent div. Here the class does nothing, so it can be removed.